src/actions/LoginAction.js

```diff
@@ -0,0 +1,43 @@
+import axios from "axios";
+
+import { toast } from "react-toastify";
+import { LOGIN_STARTED, LOGIN_FAIL, SUCCESSFUL } from "./types";
+
+export const successLogin = data => {
+  return {
+    type: SUCCESSFUL,
+    payload: data.data.user.token
+  };
+};
+
+export const userLoginRequest = (userData, props) => async dispatch => {
```

Comment:
add a statement here to close already open toasts so as to avoid having two toasts at a time. You could use `toast.dismiss()`

Reply:
Thanks, @MuhanguziDavid, I have made changes.